resources/views/content/scripts.twig

```diff
@@ -26,11 +26,9 @@
     });
 </script>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/svg-injector/1.1.3/svg-injector.min.js"></script>
-
 {% if isSearchPage or activeOnCatPage %}
-    <script src="{{ plugin_path("Findologic") }}/js/dist/filters-component-min.js"></script>
-    <!-- SSR:entry({{ ssr_entry('Findologic', 'resources/js/dist/filters-component-min.js') }}) -->
+    <script src="{{ plugin_path("Findologic") }}/js/dist/findologic-plugin-min.js"></script>
+    <!-- SSR:entry({{ ssr_entry('Findologic', 'resources/js/dist/findologic-plugin-min.js') }}) -->
```

Comment:
If we're already renaming this, please use `.min.js` instead of `-min.js`.
```suggestion
    <script src="{{ plugin_path("Findologic") }}/js/dist/findologic-plugin.min.js"></script>
    <!-- SSR:entry({{ ssr_entry('Findologic', 'resources/js/dist/findologic-plugin.min.js') }}) -->
```